### ISSUE TITLE
Version 1.12 reports as being incompatible with 1.11

### ISSUE THREAD

**Issue by ermshiperete** [jenkinsci/categorized-view-plugin#29]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
Jenkins: 2.319.2
OS: Linux - 5.4.119-1-pve
---
...
categorized-view:1.11
...
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Ubuntu 18.04.6 LTS

### Reproduction steps

The Jenkins update center shows that there is an update available for categorized-view plugin (version 1.12), but it shows a warning:

```
Warning: The new version of this plugin is marked as incompatible with the installed version. This is usually
the case because its behavior changed, or because it uses a different settings format than the installed
version. Jobs using this plugin may need to be reconfigured, and/or you may not be able to cleanly revert
to the prior version without manually restoring old settings. Consult the plugin release notes for details. 
```

However, there is no information anywhere what incompatible changes it would introduce. Looking at the difference in source code it looks more like the property `<hpi.compatibleSinceVersion>2.249.1</hpi.compatibleSinceVersion>` in `pom.xml` is wrongly initialized with the Jenkins version and should have the plugin version instead:

`<hpi.compatibleSinceVersion>1.11</hpi.compatibleSinceVersion>`

### Expected Results

no incompatibility showing, or documentation what changed that makes it incompatible

### Actual Results

see above

**Comment by Zohhak** [comment]
That was a mistake from my PR #24 back then. I didn't know all properties at that time. It was fixed with PR #26.

**Comment by ermshiperete** [comment]
That's good. Would it be possible to release a new version that contains this fix?

**Comment by basil** [comment]
Fixed in #26. Released in [1.13](https://github.com/jenkinsci/categorized-view-plugin/releases/tag/categorized-view-1.13).
